config.md

```diff
@@ -0,0 +1,157 @@
+# The js-ipfs config file
+
+The js-ipfs config file is a JSON document, which is located in a repository
+(by default located at ~/.jsipfs/config), where the repository location may be
+changed according to the value of the $IPFS_PATH environment variable.
+It is read once at node instantiation, either for an offline command, or when
+starting the daemon. Commands that execute on a running daemon do not read the
+config file at runtime.
+
```

Comment:
I'd suggest rephrasing this a little to make clear the distinction between the repo and the config file, eg:
`The js-ipfs config file is a JSON document located in the root directory of the js-ipfs repository (by default the js-ipfs repository is located at `~/.jsipfs` and the config file is at `~/jsipfs/config`). The repository location may be changed with the `$IPFS_PATH` environment variable. The config file is read once when the js-ipfs daemon is started, or each time a command is executed in offline mode. Commands that execute on a running daemon do not read the config file at runtime.`

Reply:
Should we also make a distinction for the config file inside the browser? I am trying to avoid getting really complex in the introduction part.

Reply:
Oh hmm that's true. I guess this isn't really about the ipfs repo per se, it's more about the config file itself. Maybe better to just talk about the config file to keep it simple, something like:
`The js-ipfs config file is a JSON document located in the root directory of the js-ipfs repository`

Reply:
Agreed! Just put it like that